helpers/termux_dependencies.py

```diff
@@ -4,19 +4,21 @@
 def InstallTermuxDependencies() : 
     CustomPrint("Installing dependencies...")
     CustomPrint("Updating Termux")
-    os.system('apt update')
-    CustomPrint("Allow storage permission for storing extraced whatsapp.ab in interal storage.")
+    os.system('pkg update && pkg upgrade')
+    CustomPrint("Allow storage permission for storing extracted whatsapp.ab in interal storage:")
     os.system('termux-setup-storage')
-    CustomPrint("Done. Installing various required components.")
-    os.system('apt install curl grep tar proot wget -y')
-    CustomPrint("Done. Installing ADB for Termux")
+    CustomPrint("Done. Installing required dependencies...")
+    os.system('pkg install curl grep tar proot wget -y')
+    os.system('pip install termcolor')
     os.system('wget https://github.com/MasterDevX/Termux-ADB/raw/master/InstallTools.sh && bash InstallTools.sh')
-    CustomPrint("Done. Installing Java for termux.")
     try : 
-        os.system('rm -rf installjava') # Deleting any previous instance of installjava.
+        os.system('rm -r -f installjava') # Deleting any previous instance of installjava.
     except Exception as e : 
         pass
-    os.system('wget https://raw.githubusercontent.com/Hax4us/java/master/installjava && sh installjava')
-    CustomPrint("Done installing all dependencies. Now running proot.")
+    os.system('wget https://raw.githubusercontent.com/MasterDevX/java/master/installjava && sh installjava')
+    os.system('proot login')
```

Comment:
Proot login should be run every time script is run and not just when getting dependencies.

Reply:
You may add that in documentations later on. Merging anyway.